src/services/tags/src/controllers/tags.ts

```diff
@@ -40,11 +42,13 @@ export default class TagsController {
 		}
 	}
 
-	private static getParsedTagData(tags: Array<iTag>): Array<object> {
+	private static getParsedTagData(tags: Array<iTag>, questionTags: Array<iQuestionTag>): Array<object> {
```

Comment:
getParsedTagData should return an array of Object with defined interface not generic 'object'